src/components/TextFieldChip/TextFieldChips.tsx

```diff
@@ -190,6 +167,51 @@ const TextFieldChips = React.forwardRef(
       })
     }
 
+    const handleClickAway = () => {
+      if (!isFocusingRef.current) {
+        return
+      }
+      if (chipIndexEditable !== null) {
+        clearChipIndexEditable()
+        clearInputValue()
+      } else if (addOnBlur) {
+        const inputValueTrimed = inputValue.trim()
+        if (inputValue.length > 0) {
+          if (inputValueTrimed.length === 0) {
+            clearInputValue()
+          } else if (chipIndexEditable !== null) {
+            updateChip(inputValueTrimed, chipIndexEditable)
+          } else {
+            addChip(inputValueTrimed)
+          }
+        } else if (
```

Comment:
I don't get it, you remove chip on Blur ? why ?

Reply:
Ok you copied the code from handleKeyDown, but the feature here is only adding a chip, or update one on blur, that's all.

Reply:
Removed that block 👍